src/Promotions/InPluginUpsells/Endpoints/ProductRecommendationsRoute.php

```diff
@@ -114,6 +115,10 @@ public function handleRequest(WP_REST_Request $request)
             case 'givewp_donors_fee_recovery_recommendation_dismissed':
                 update_option('givewp_donors_fee_recovery_recommendation_dismissed', time());
                 break;
+
+            case 'givewp_form_editor_donation_options_recurring_recommendation' :
```

Comment:
For what it is worth, since the value of the `option` param is already validated as an `enum`, this `switch` statement could be removed in favor of passing the value of `option` directly to `update_option`.

Reply:
Much nicer! Thanks https://github.com/impress-org/givewp/pull/6802/commits/53e12b66c4ad4326df7fd7607b190af36e16e15d